Tests/DatadogTests/Datadog/Tracing/Utils/ActiveSpanUtilsTest.swift

```diff
@@ -0,0 +1,94 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import XCTest
+@testable import Datadog
+
+class ActiveSpanUtilsTest: XCTestCase {
+    override func tearDown() {
+        //Confirm that tests have cleaned up spans
+        XCTAssertNil(ActiveSpanUtils.getActiveSpan())
+        super.tearDown()
+    }
+
+    func testsWhenSpanIsStartedIsAssignedToActiveSpan() throws {
+        let tracer = Tracer.mockAny()
+        let previousSpan = ActiveSpanUtils.getActiveSpan()
```

Comment:
We could use `tracer.activeSpan()` instead of `ActiveSpanUtils.getActiveSpan()` in all this tests. This way we're also testing the public API, instead of internals.

Reply:
Yes, changing